### PR TITLE
Removes comments from schema tokens

### PR DESCRIPTION
**Pull request by toddlees**

### CODE REVIEW

**Review by mjtravers — CHANGES_REQUESTED** [review]
Don't forget to run the generate-schema-doc for the SchA and copy the HTML output to the /docs directory

**Comment by toddlees** [comment]
I've reverted the backlog schema, but it's worth noting that running the generate schema docs script without specifying a file results in a SchA.html based on schema/backlog/SchA.json rather than schema/SchA.json, which might be unexpected.

File: schema/SchA.json [hunk 1]
```diff
@@ -436,7 +436,7 @@
                 "FIELD_FORM_ASSOCIATION": "F3 | F3X | F3P"
             }
         },
-        "contribution_amount_{f3l_bundled}": {
+        "contribution_amount": {
```
Comment (mjtravers) [inline schema/SchA.json:439]: The property name in the fec_recommended array needs to be removed as well:

    "fec_recommended": [
        "contributor_street_1",
        "contributor_city",
        "contributor_state",
        "contributor_zip",
        "contribution_date",
        "contribution_amount_{f3l_bundled}"
    ],

**Review by mjtravers — CHANGES_REQUESTED** [review]
Found a {} reference in the fec_recommended array. We'll have to be sure to no longer copy files from the backlog directory up to the schema directory now that we're editing the schema files directly.

**Review by mjtravers — APPROVED** [review]
looks good

**Pull request merged**